src/components/AddToList/index.js

```diff
@@ -96,7 +96,13 @@ function Alert(props) {
   return <MuiAlert elevation={6} variant="filled" {...props} />;
 }
 
-function AddToList({ handle, results: listsProp, ...props }) {
+function AddToList({
+  handle,
+  results: listsProp,
+  label,
```

Comment:
```suggestion
  addToListLabel,
```
or `addLabel`... but not just `label`.